### ISSUE TITLE
Add /dist folder to .gitignore

### ISSUE THREAD

**Issue by pyrello** [uiowa/uids#100]
Similar to USWDS: https://github.com/uswds/uswds/blob/develop/.gitignore#L14

**Comment by pyrello** [comment]
This is maybe related to #73

**Comment by richardbporter** [comment]
I'm not sure we can ignore dist/ since the uids_base theme uses that. I can look into changing the build and release process to make it a bit easier.

**Comment by richardbporter** [comment]
This feels like it should be a part of #79 along with the folder restructure. However, there may be a fix in the short term. Thoughts?

**Comment by pyrello** [comment]
@richardbporter The idea with adding `/dist` to .gitignore is that it is automatically generated by running `npm run build`, so presumably, it would get built as part of our automated build process. Not that it would be removed from the project as a resource.

**Comment by richardbporter** [comment]
Our current automated build process pushes to the gh-pages page, basically for documentation purposes. Yarn/NPM get the package information from the tags created off master. If we ignore dist/ and build/ on master, they won't be included in the package.

I can take a closer look at this at some point but right now https://github.com/uiowa/uiowa/issues/1281 is my priority.

**Comment by pyrello** [comment]
Does this line not run a build process that creates the `/dist` folder? https://github.com/uiowa/uids/blob/master/.travis.yml#L15

**Comment by pyrello** [comment]
I have an idea for how to handle this. It would involve a new npm command to create a new release. Currently, we run an npm command: `npm version {major|minor|patch} -m "Message"`. This new command could run the build process and include the build directory, bump the version and tag a new release. This would be the "official" way of creating a new release that insured that the "/dist" folder was always included.

**Comment by pyrello** [comment]
Here is an example of a release gulp task in USWDS: https://github.com/uswds/uswds/blob/develop/config/gulp/release.js

It looks like they are using CircleCI instead of Travis, but the format seems pretty similar. That command gets called as part of their deployment process: https://github.com/uswds/uswds/blob/develop/.circleci/config.yml#L61

**Comment by pyrello** [comment]
It looks like their git workflow is a little different than ours: https://github.com/uswds/uswds/wiki/Release-process#git-workflow

In their workflow, the `master` branch is only updated at the point when a new release is created. They don't seem to use a "stable but not released" branch (our `master`). It looks like `develop` is more like our master branch.

**Comment by bspeare** [comment]
@pyrello is this still an issue?

**Comment by pyrello** [comment]
Yes, this is still relevant. I have updated the IS and added a label to target the 4.x branch.

**Comment by pyrello** [comment]
Any of the comments before this one may be out of date: https://github.com/uiowa/uids/issues/100#issuecomment-976848572
